pkg/template/registry/rest.go

```diff
@@ -47,10 +47,15 @@ func (s *REST) Create(ctx kapi.Context, obj runtime.Object) (runtime.Object, err
 		"expression": generator.NewExpressionValueGenerator(rand.New(rand.NewSource(time.Now().UnixNano()))),
 	}
 	processor := template.NewProcessor(generators)
-	if errs := processor.Process(tpl); len(errs) > 0 {
+	errs, err := processor.Process(tpl)
```

Comment:
maybe use different var names to distinguish the types of error you're expecting returned? (templateErrors, err)